airbyte-cdk/python/airbyte_cdk/sources/streams/core.py

```diff
@@ -89,16 +100,17 @@ def get_error_display_message(self, exception: BaseException) -> Optional[str]:
 
     @abstractmethod
     def read_records(
```

Comment:
the method name is a little misleading now because it doesn't only return records anymore